engine/src/flutter/lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -170,7 +170,10 @@ void _insertEditingElementInView(DomElement element, int viewId) {
     view != null,
     'Could not find View with id $viewId. This should never happen, please file a bug!',
   );
-  view!.dom.textEditingHost.append(element);
+  final host = view!.dom.textEditingHost;
+  if (!host.contains(element)) {
+    host.append(element);
+  }
```

Comment:
Oh, this prevents blurs because the element gets "moved" in the DOM when already inserted? I think this deserves a small comment so it doesn't get "optimized" in the future :P

Reply:
Exactly. Yes a comment is in order.